views/partials/footer.hbs

```diff
@@ -1,3 +1,3 @@
-<footer class="f6 tc">
+<footer class="f6 tc footer">
```

Comment:
You can just use `flex-none` here https://github.com/tachyons-css/tachyons-flexbox